### PR TITLE
feat: Add profile verification

### LINKED ISSUE #1887: Add profile verification
**Describe the feature you'd like**
- Display verification status in profile
- Provide option to email verification link

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #1887 

<p align="center">
 
 
</p>

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Change verified icon and text to green

**Comment by liveHarshit** [comment]
> Change verified icon and text to green

Updated

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -77,6 +77,9 @@
     <string name="edit_profile_photo_message">Edit your profile picture</string>
     <string name="delete">Delete</string>
     <string name="new_photo">New Photo</string>
+    <string name="verified">Verified</string>
+    <string name="not_verified">User not verified</string>
```
Comment (nikit19) [inline app/src/main/res/values/strings.xml]: I guess something like "Your account hasn't been verified" would be better

Comment (liveHarshit) [reply app/src/main/res/values/strings.xml]: But too long. The layout will take two lines.

Comment (nikit19) [reply app/src/main/res/values/strings.xml]: Resend verification mail can be in the next line?

Comment (liveHarshit) [reply app/src/main/res/values/strings.xml]: Updated

**Pull request merged**